measure_extinction/extdata.py

```diff
@@ -534,7 +555,7 @@ def trans_elv_alav(self, av=None, akav=0.112):
                 self.exts[curname] = (self.exts[curname] / av) + 1
                 self.uncs[curname] /= av
             # update the extinction curve type
-            self.type = "alav"
+            self.type = "alax"
```

Comment:
I've been changing all 'alav' to 'alax' when I'm working on the code.  The alax is what it should be with rel_band=V.  But you are right I am not doing this everywhere in the code.  Not quite sure the right way to go here as some places have been changed.  I think I'll go ahead and change things and then work to make the updates elsewhere in a different pull request.  At least if things break, it will force me (or us) to make the alav -> alax updates.